rfc6962-bis.xml

```diff
@@ -946,47 +956,52 @@ an accepted root certificate.
-	      </t>
-            </list></t>
-          </list>
-	</t>
-	<t>
-          Both tree sizes must be from existing v1 STHs (Signed Tree Heads). However, because of skew, the receiving front-end may not know one or both of the existing STHs. If both are known, then only the <spanx style="verb">consistency</spanx> output is returned. If the first is known but not the second, then the latest known STH is returned, along with a consistency proof between the first STH and the latest. If neither are known, then the latest known STH is returned without a consistency proof.
-        </t>
-        <t>
-          Outputs:
-
-        <list>
-	  <t><list style="hanging">
-            <t hangText="consistency:">
-	      An array of Merkle Tree nodes, base64 encoded.
-            </t>
-	    <t hangText="tree_size:">
-	      The size of the tree, in entries, in decimal.
-	    </t>
-	    <t hangText="timestamp:">
-	      The timestamp, in decimal.
-	    </t>
-	    <t hangText="sha256_root_hash:">
-	      The Merkle Tree Hash of the tree, in base64.
+	  <list style="hanging">
+            <t hangText="Inputs:">
+	      <list style="hanging">
+		<t hangText="first:">
+		  The tree_size of the older tree, in decimal.
+		</t>
+		<t hangText="second:">
+		  The tree_size of the newer tree, in decimal.
+		</t>
+              </list>
 	    </t>
-	    <t hangText="tree_head_signature:">
-	      A TreeHeadSignature for the above data.
+	    <t>
+              Both tree sizes must be from existing v1 STHs (Signed Tree Heads). However, because of skew, the receiving front-end may not know one or both of the existing STHs. If both are known, then only the <spanx style="verb">consistency</spanx> output is returned. If the first is known but not the second, then the latest known STH is returned, along with a consistency proof between the first STH and the latest. If neither are known, then the latest known STH is returned without a consistency proof.
+            </t>
+            <t hangText="Outputs:">
+	      <list style="hanging">
+		<t hangText="consistency:">
+		  An array of Merkle Tree nodes, base64 encoded.
+		</t>
+		<t hangText="tree_size:">
+		  The size of the tree, in entries, in decimal.
+		</t>
+		<t hangText="timestamp:">
+		  The timestamp, in decimal.
+		</t>
+		<t hangText="sha256_root_hash:">
+		  The Merkle Tree Hash of the tree, in base64.
+		</t>
+		<t hangText="tree_head_signature:">
+		  A TreeHeadSignature for the above data.
+		</t>
+              </list>
+              Note that no signature is required on this data, as it is used to verify an STH, which is signed.
+            </t>
+	    <t hangText="Errors:">
+	      <list style="hanging">
+		<t hangText="first unknown">
+		  <spanx style="verb">first</spanx> is before the latest known STH but is not from an existing STH.
+		</t>
+		<t hangText="second unknown">
+		  <spanx style="verb">second</spanx> is before the latest known STH but is not from an existing STH.
```

Comment:
Do you get the error *in addition* to the latest STH in case the second sth provided is unknown? the description of the method makes it clear than in case of a known first sth but unknown second sth the latest sth (+ consistency proof) is returned. no mentions of error.